src/components/LessonInfoDisplay/LessonInfoDisplay.js

```diff
@@ -2,15 +2,21 @@ import React, { useState } from "react";
 import { Col, Row } from "reactstrap";
```

Comment:
general comment- are both PRs up to date w each other? is there a preference for which one i should be making comments on/merging in?

Reply:
i think it would be best to merge "delete" first because this isn't updated with any changes that needed to be made. So once thats merged, Ill merge from master and then this one will be ready

Reply:
right now both are up to date with each other. The only difference should be the delete-confirmation doesnt have the back button